Gemfile

```diff
@@ -37,7 +37,11 @@ group :development do
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
   gem 'spring-watcher-listen', '~> 2.0.0'
-  gem 'rspec-rails', '~> 4.0.1'
```

Comment:
Adding this to the :development group is not strictly necessary, but without it generators and rake tasks must be preceded by RAILS_ENV=test

Reply:
But it's in `group :development, :test do`? Thats the same as having it in group :development & group :test? Should be available in dev env's.

What's a task to run? Ill test if it needs it rails_env to be test.